content/en/blog/_posts/2020-04-22-Supporting-the-Evolving-Ingress-Specification-in-Kubernetes-1.18.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: Supporting the Evolving Ingress Specification in Kubernetes 1.18
+date: 2020-04-22
+slug: Supporting-the-Evolving-Ingress-Specification-in-Kubernetes-1.18
+---
+
+**Authors:** Alex Gervais (Datawire.io)
+
+Last month, the Kubernetes team released [Kubernetes 1.18](https://kubernetes.io/blog/2020/03/25/kubernetes-1-18-release-announcement/), which extended Ingress. In this blog post, we’ll walk through what’s new in the new Ingress specification, what it means for your applications, and how to upgrade to an ingress controller that supports this new specification.
+
+### What is Kubernetes Ingress
+When deploying your applications in Kubernetes, one of the first challenges many people encounter is how to get traffic into their cluster. [Kubernetes ingress](https://blog.getambassador.io/kubernetes-ingress-nodeport-load-balancers-and-ingress-controllers-6e29f1c44f2d) is a collection of routing rules that govern how external users access services running in a Kubernetes cluster. There are three general approaches for exposing your application:
+
+* Using a `NodePort` to expose your application on a port across each of your nodes
+* Using a `LoadBalancer` service to create an external load balancer that points to a Kubernetes service in your cluster
+* Using a Kubernetes Ingress Resource
+
+### What’s new in Kubernetes 1.18 Ingress
+There are three significant additions to the Ingress API in Kubernetes 1.18:
+
+* A new `pathType` field
+* A new `IngressClass` resource
```

Comment:
We'd usually write this:
````suggestion
* A new IngressClass resource
```

Reply:
Is there any reason why IngressClass would not be formated when `pathType` would?